src/dotnet/APIView/APIViewWeb/wwwroot/css/StyleSheet.css

```diff
@@ -0,0 +1,27 @@
+﻿body {
```

Comment:
Add contents of this file to site.css, no reason to have multiple css files yet.